platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformCachingIntegrationTest.groovy

```diff
@@ -2015,7 +2015,7 @@ class ArtifactTransformCachingIntegrationTest extends AbstractHttpDependencyReso
                     String postfix = identifier.map { it -> " " + it }.getOrElse("")
                     println "files\${postfix}: " + artifacts.artifactFiles.collect { it.name }
                     artifacts.artifactFiles.each { assert it.exists() }
-                    println "ids\${postfix}: " + artifacts.collect { it.id.displayName }
+                    println "artifacts\${postfix}: " + artifacts.collect { it.file.name + " (" + it.id.componentIdentifier + ")" }
```

Comment:
🤔 Couldn't we just change `DefaultResolvedArtifactResult` to do something like this?

Would it be useful to include all the information (file name, IvyArtifactName and componentId)?

Reply:
Discussed. We could introduce a `getDisplayName` on `ArtifactResult`, but we decided its not worth it for now